### PR TITLE
🎉 New Source: the-forex-api

### PR DESCRIPTION
**Pull request by olivermeyer**
## What
Creating a new source for `the-forex-api` using the low-code CDK.

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [x] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

### CODE REVIEW

**Comment by olivermeyer** [comment]
Various integration tests are failing. Some of them expect a `spec.yaml` file, but my understanding is that it's not required since the spec is now in the `manifest.yaml`. `test_read` is failing with the following error
```
    @staticmethod
    def check_datetime(value: str) -> bool:
>       valid_format = timestamp_regex.match(value)
E       TypeError: expected string or bytes-like object

../../bases/source-acceptance-test/source_acceptance_test/utils/asserts.py:30: TypeError
-----------------------------------------------------------------------------------------
```
I'm not sure what it's trying to do. I am able to run the connector locally.

I'd appreciate some help in getting the tests to pass :)

**Comment by olivermeyer** [comment]
@marcosmarxm can you help out? Not sure who I need to tag nowadays :)

File: airbyte-integrations/connectors/source-the-forex-api/source_the_forex_api/schemas/rates.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "properties": {
+    "date": {
+      "type": ["null", "string"]
+    },
+    "base": {
+      "type": ["null", "string"]
+    },
+    "rates": {
+      "type": ["null", "string"],
+      "format": "date-time"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-the-forex-api/source_the_forex_api/schemas/rates.json:13]: Can you try to remove this format from rates?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Comment by olivermeyer** [comment]
@marcosmarxm thanks for taking a look at this! Unfortunately the API has been down since Friday, so we're going to find a more stable source for this data.

**Pull request closed without merging**